app/Template/layout.php

```diff
@@ -50,6 +50,7 @@
           data-timezone="<?= $this->app->getTimezone() ?>"
           data-js-date-format="<?= $this->app->getJsDateFormat() ?>"
           data-js-time-format="<?= $this->app->getJsTimeFormat() ?>"
+          <?php if ($this->app->isRTL($this->app->jsLang())) {echo('dir="RTL"');} ?>
```

Comment:
HTML has `dir="auto"` attribute that might simplify this.

https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/dir

Reply:
In this case, it doesn't. `dir="auto"` works with first strong character on an element. This would work for individual elements (on some of which I should add this) but not necessarily on body tag. Here my intention was to force RTL for layout if the language is one of RTL languages like Farsi (Persian) or Arabic.

Reply:
So far, everything looks neat. There is only one minor issue for markdown editor which wouldn't occur in regular situations. t should be resolved at the level of markdown parser (by adding `dir="auto"` to each direct child element). I am not sure if we can do it without modification in parser or using js for DOM manipulation.

The other issue is with one class which is hardcoded for floating to right side. Not sure what to do with it. Should I change the class name and then its styles?

When do you think this PR would be ready for merge (if it would be at all)?